### PR TITLE
Improve coverage

### PR DESCRIPTION
**Pull request by fabwu**
My first try to improve coverage. Is this way correct?

### CODE REVIEW

**Comment by ojhaujjwal** [comment]
Yes. this is how it is done. See that was easy.

**Comment by ojhaujjwal** [comment]
Can you improve test coverage for `HtUserRegistration\Service\UserRegistrationService`?

**Comment by fabwu** [comment]
Next I will improve `UserRegistrationMapper`. I'm looking forward to your feedback.

**Comment by ojhaujjwal** [comment]
I am celebrating holidays and I will get back to this when I am free.

File: composer.json [hunk 1]
```diff
@@ -1,34 +1,37 @@
-{
```
Comment (ojhaujjwal) [inline composer.json:1]: @wuethrich44  Why is github showing that entire composer.json is changed? How am I going to do code review like this? I can't find the specific changes that you have made. Can you fix this?

File: src/Service/UserRegistrationService.php [hunk 2]
```diff
@@ -1,223 +1,204 @@
-<?php
```
Comment (ojhaujjwal) [inline src/Service/UserRegistrationService.php:1]: Same problem here. The entire contents of file is changed.

**Comment by fabwu** [comment]
I had wrong endings configured on my windows machine. I move the tests into a new proper branch #17

**Pull request closed without merging**